system/security/LdapSecurity/aci.cpp

```diff
@@ -654,6 +654,7 @@ class CAciList : implements IInterface, public CInterface
         }
 
         int perm = 0;
```

Comment:
is perm meant to be a SecAccessFlags ?

Update: Noticed bit-wise operations are performed on "perm", can we avoid bitwise ops and use enum logic instead?